bokeh/core/property/container.py

```diff
@@ -356,6 +356,24 @@ def __init__(self, default={}, help=None):
     def __str__(self):
         return self.__class__.__name__
 
+class RestrictedDict(Dict):
+    """ Check for disallowed key(s).
+
+    """
+
+    def __init__(self, keys_type, values_type, disallow, default={}, help=None):
+        self._disallow = set(disallow)
+        super().__init__(keys_type=keys_type, values_type=values_type, default=default, help=help)
+
+    def validate(self, value, detail=True):
+        super().validate(value, detail)
+
+        error_keys = self._disallow & value.keys()
+
+        if not error_keys:
```

Comment:
this condition is reversed, also need to follow the pattern for `msg` and `detail` I pointed out earlier (see other examples in the same file)